File: array/array.f90
! array.f90
program ArrayExample
    implicit none
    integer :: arr(5) = [1, 2, 3, 4, 5]
    integer :: i

    print *, "Array elements are: "
    do i = 1, 5
        print *, arr(i)
    end do
end program ArrayExample

File: array/array.pas
// array.pas
program ArrayExample;

var
    arr: array[1..5] of integer = (1, 2, 3, 4, 5);
    i: integer;

begin
    Write('Array elements are: ');
    for i := 1 to 5 do
        Write(arr[i], ' ');
    WriteLn;
end.
